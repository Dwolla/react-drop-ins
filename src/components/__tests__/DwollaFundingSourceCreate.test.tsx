import TestRenderer from 'react-test-renderer';
import { DwollaFundingSourceCreate } from '../index';

describe('DwollaFundingSourceCreate', () => {
  it('should render component without micro-deposit initiation', () => {
    const tree = TestRenderer.create(<DwollaFundingSourceCreate customerId="123" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render component with micro-deposit initiation', () => {
    const tree = TestRenderer.create(
      <DwollaFundingSourceCreate customerId="123" initiateMicroDeposits />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
