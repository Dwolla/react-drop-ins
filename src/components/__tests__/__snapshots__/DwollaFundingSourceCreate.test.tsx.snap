// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaFundingSourceCreate should render component with micro-deposit initiation 1`] = `
<dwolla-funding-source-create
  customerId="123"
  initiateMicroDeposits={true}
/>
`;

exports[`DwollaFundingSourceCreate should render component without micro-deposit initiation 1`] = `
<dwolla-funding-source-create
  customerId="123"
/>
`;
